docker/scripts/grub-installer.sh

```diff
@@ -126,7 +126,7 @@ install_grub_osie() {
 	else
 		[[ $arch == aarch64 ]] && mount -o remount,ro /sys/firmware/efi/efivars
 
-		grub-install --recheck --bootloader-id=GRUB --root-directory="$target" --efi-directory="$target/boot/efi"
+		grub-install --recheck --bootloader-id=ubuntu --root-directory="$target" --efi-directory="$target/boot/efi"
```

Comment:
Since it seems like the bootloader_id has to exactly match between here and the later `efibootmgr` line later - could you please extract it into a variable?

`readonly BOOTLOADER_ID=ubuntu`

Reply:
Not sure if this is going to stay this way to get centos (and presumably debian and others) working. I'll get rid of the duplication if it happens to stick around though.